packages/compat/legacy/class.jetpack-client.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * Class Jetpack_Client
+ *
+ * @deprecated Use Automattic\Jetpack\Connection\Client
+ */
+class Jetpack_Client {
+
+	/**
+	 * @deprecated use Automattic\Jetpack\Connection\Client::remote_request
+	 */
+	static function remote_request( $args, $body = null ) {
+		return Client::remote_request( $args, $body );
```

Comment:
Let's throw one of these https://developer.wordpress.org/reference/functions/_deprecated_function/